### PR TITLE
Swipes for columns

### PR DESCRIPTION
**Pull request by rauchp**
showcasing the mid transition since it's what we really gained from this. Swipe works.

Simulator Screen Shot - iPhone 11 - 2019-10-13 at 20 30 25
Simulator Screen Shot - iPhone 11 - 2019-10-13 at 20 21 44

### CODE REVIEW

**Comment by rauchp** [comment]
why ted cruz looking like a deadass neckbeard tho

File: civilapp/profile/ProfileView.js [hunk 1]
```diff
@@ -182,17 +181,31 @@ class ProfileView extends React.Component {
...
 
-  setFollowing = () => {
-    // set userlist in state to subscribers
-    this.setState({ showFollowers: false })
-  }
 
-  setFollowers = () => {
-    this.setState({ showFollowers: true })
+  renderUserList = (data) => {
+    return <FlatList style={{ width: '100%', flex: 1 }}
```
Comment (EphemeralX) [inline civilapp/profile/ProfileView.js]: small nit: wrap this in `()`

File: civilapp/profile/ProfileView.js [hunk 2]
```diff
@@ -182,17 +181,31 @@ class ProfileView extends React.Component {
...
+      keyExtractor={item => item.username + "" + Math.random()}
+      renderItem={(it) => {
```
Comment (EphemeralX) [inline civilapp/profile/ProfileView.js]: small nit: rename `it` to something more informative so we know what kind of object to expect (userInfo? userPreview?)

File: civilapp/yarn.lock [hunk 3]
```diff
@@ -6461,7 +6470,7 @@ react-test-renderer@^16.8.6:
     react-is "^16.8.6"
     scheduler "^0.13.6"
 
-react-timer-mixin@^0.13.4:
+react-timer-mixin@^0.13.3, react-timer-mixin@^0.13.4:
```
Comment (EphemeralX) [inline civilapp/yarn.lock]: What's this?

File: civilapp/search/SearchScreen.js [hunk 4]
```diff
@@ -189,8 +167,19 @@ class SearchScreen extends Component {
           disableRight
           borderWidth={0}
         />
-        {this.renderTypeHeader()}
-        {showRooms ? this.renderRooms() : this.renderUsers()}
+        {this.renderSearchInput()}
+        <ScrollableTabView
+            onChangeTab={(i)=> this.setState({ showRooms: (i==0)})}
```
Comment (EphemeralX) [inline civilapp/search/SearchScreen.js]: nit: `i` -> `index` (assuming that's what it stands for)

**Comment by rauchp** [comment]
https://drive.google.com/file/d/1-9Hw48nxX6ZeFUPRPaeTGcr7KWvSjPM4/view?usp=sharing

added new swipe

File: civilapp/yarn.lock (hunk 3 above)
Comment (rauchp) [reply civilapp/yarn.lock]: dependency hell

**Comment by rauchp** [comment]
also I don't see a merge conflict?

File: civilapp/current/UserModal.js [hunk 5]
```diff
@@ -293,35 +294,52 @@ const MenuButton = props => {
...
+          props.swipeLeft()
+        }
+      }}>
+      <FlingGestureHandler
+        direction={Directions.RIGHT}
+        onHandlerStateChange={({ nativeEvent }) => {
+          if (nativeEvent.state === State.ACTIVE) {
+            props.swipeRight()
+          }
+        }}>
```
Comment (EphemeralX) [inline civilapp/current/UserModal.js]: Seems a little weird to a FlingGestureHandler inside another, but if it works, no complaints.

Comment (rauchp) [reply civilapp/current/UserModal.js]: https://stackoverflow.com/a/8813202

**Pull request merged**